starter-repos/app-template/skaffold/build.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+skaffold-build:
+  stage: build
+  image: gcr.io/k8s-skaffold/skaffold:v1.12.0
```

Comment:
nit: `v1.14.0`

Reply:
oh, yeah. I used exactly the same image as golang-template to make sure the build works. Not sure if v1.14.0 could have any issues, I'll do an e2e test once Haiyan's AR script is available.